modules/transport/http/src/org/apache/axis2/transport/http/HTTPSender.java

```diff
@@ -323,45 +322,12 @@ private void handleResponse(MessageContext msgContext,
                 }
             }
         }
-        if (HTTPStatusCodeFamily.SUCCESSFUL.equals(family)) {
+        if (!errorCodes.contains(statusCode)) {
             // Save the HttpMethod so that we can release the connection when cleaning up
             // HTTP 202 Accepted should support body based on ESBJAVA-4370 as spec does not strictly enforce
             // no-entity-body with 202 Accepted response.
```

Comment:
Is the above comment still valid?

Reply:
Removed the comment in [8146236](https://github.com/wso2/wso2-axis2/pull/166/commits/814623632e32e24d87c1195971fd45302eaeaace) as it becomes invalid after the fix.